src/13_file_io.py

```diff
@@ -10,10 +10,23 @@
 # Note: pay close attention to your current directory when trying to open "foo.txt"
 
 # YOUR CODE HERE
+with open('./foo.txt', 'r') as f:
+    read_data = f.read()
+    print(read_data)
 
 # Open up a file called "bar.txt" (which doesn't exist yet) for
 # writing. Write three lines of arbitrary content to that file,
 # then close the file. Open up "bar.txt" and inspect it to make
 # sure that it contains what you expect it to contain
 
-# YOUR CODE HERE
\ No newline at end of file
+# YOUR CODE HERE
+
+# I tried to write and read at the same time with 'r+' mode but couldn't. Can we review this?
+with open('./bar.txt', 'w') as f:
+    f.write("You should be added to the file now. Are you?\n")
+    f.write("Second line.\n")
+    f.write("The final third line.\n")
```

Comment:
Your code looks good. Double-check if you are on the correct directory in the terminal before you run the file.

Reply:
Found 'w+' mode to work for creating the file, writing lines, and reading it afterwards.